src/group/errors.rs

```diff
@@ -108,3 +113,11 @@ impl From<TreeError> for WelcomeError {
         }
     }
 }
+
+impl From<HpkeError> for WelcomeError {
```

Comment:
Shouldn't we have an `HpkeError(HpkeError)` in the enum instead? (Possible with #274 landed)

Reply:
You mean `WelcomeError(HpkeError)`?

Reply:
yeah... 🙈